empress_gui.py

```diff
@@ -4,362 +4,819 @@
+
+        self.recon_info_displayed = False
+        
+        if self.cost_space_window is not None and self.cost_space_window.winfo_exists() == 1:
+            self.cost_space_window.destroy()
+
+        if self.entire_space_window is not None and self.entire_space_window.winfo_exists() == 1:
+            self.entire_space_window.destroy()
+        
+        if self.set_num_cluster_window is not None and self.set_num_cluster_window.winfo_exists() == 1:
+            self.set_num_cluster_window.destroy()
+
+        if self.view_solution_space_window is not None and self.view_solution_space_window.winfo_exists() == 1:
+            self.view_solution_space_window.destroy()
+
+        if self.one_MPR_window is not None and self.one_MPR_window.winfo_exists() == 1:
+            self.one_MPR_window.destroy()
+
+        if self.view_reconciliations_window is not None and self.view_reconciliations_window.winfo_exists() == 1:
+            self.view_reconciliations_window.destroy()
+
+        if self.view_pvalue_histogram_window is not None and self.view_pvalue_histogram_window.winfo_exists() == 1:
+            self.view_pvalue_histogram_window.destroy()
+
+    def load_input_files(self, event):
+        """Load in two .nwk files for the host tree and parasite tree, and one .mapping file. Display the number of tips for 
+        the trees and a message to indicate the successful reading of the tips mapping.""" 
+        # Clicking on "Load host tree file" 
+        if self.load_files_var.get() == "Load host tree file":
+            # initialdir is set to be the current working directory
+            input_file = tk.filedialog.askopenfilename(initialdir=os.getcwd(), title="Select a host file")
+            if Path(input_file).suffix == '.nwk':
+                # Try to read in host tree file
+                self.recon_input.read_host(input_file)
+                self.host_file_path = None
+                if self.recon_input.host_dict is not None:
+                    self.host_file_path = input_file
+                    # Force a sequence of loading host tree file first, and then parasite tree file, and then mapping file
+                    self.load_files_dropdown['menu'].entryconfigure("Load parasite tree file", state = "disabled")
+                    self.load_files_dropdown['menu'].entryconfigure("Load mapping file", state = "disabled")
+                    # Reset everything every time user successfully loads in a new host tree file
+                    self.reset("Load host tree file")      
+                    host_tree_tips_number = self.compute_tree_tips("host tree")
+                    self.host_tree_info = tk.Label(self.input_info_frame, text="Host: "+os.path.basename(self.host_file_path)+": "+str(host_tree_tips_number)+" tips")
+                    self.host_tree_info.grid(row=0, column=0, sticky="w")
+                else: 
+                    messagebox.showinfo("Warning", "The input file cannot be read.")          
```

Comment:
Show error/warning information in the `except` block using information from `e`.

Reply:
The current error message is not very helpful. For example, if .nwk file is expected and a .mapping file is loaded, the error message is: can only concatenate str (not "tuple") to str. If the file cannot be read, the error message is: Number of open/close parentheses do not match.